microsoft-365/admin/manage/manage-plugins-for-copilot-in-integrated-apps.md

```diff
@@ -30,7 +30,10 @@ description: "Learn how to manage plugins for Copilot in the Integrated apps pag
 # Manage Plugins for Copilot in Integrated Apps
 
 > [!IMPORTANT]
-> You must be licensed with Microsoft 365 Copilot to use plugins for Copilot for Microsoft 365. During the Public Preview, admins will need to enable their tenant by contacting their Microsoft Customer Success Account Manager or submitting a support ticket requesting to "Opt-in to plugins for Copilot for Microsoft 365."
+> 
+> - You must be licensed with Microsoft 365 Copilot to use plugins for Copilot for Microsoft 365.
+> - The capability is enabled by default in all Microsoft 365 Copilot licensed tenants.
+> - Admins can disable this functionality on a user and group basis and control how individual plugins are approved for use, and which plugins are enabled."
```

Comment:
The rest of the document covers this last point. Should we include it as a note?

Reply:
I'd include it as per the guidelines from Kalyan. 

Thank you Abram.